components/standings/commons/standings_storage.lua

```diff
@@ -152,6 +152,7 @@ function StandingsStorage.toScoreBoardEntry(data)
 
 	-- Using Table.map to filter. Because strangely enough Table.filter has no access to keys...
 	local scoreBoard = Table.mapValues(Table.map(data, filterScoreBoard), tonumber)
+	---@cast scoreBoard {w: number, d: number, l: number}
```

Comment:
```suggestion
	local scoreBoard = Table.mapValues(filterByKey(data, filterScoreBoard), tonumber)
```
+ Update the `filterScoreBoard` to return true/false instead

Reply:
will try, but don't really see how that solves the anno warning?

Reply:
It does :)

Reply:
ah didn't see the filterbykey (which also needs the `Table.`)